### DISCUSSION TITLE
Mail directory encryption

### DISCUSSION THREAD

**Discussion by asakura42** [neomutt/neomutt#4505] [Development]
## What would you like to be able to do?

I think that encryption of mail directory is good idea for security. Imagine: you use neomutt on shared server and while you properly set up ssh connection, server admin may still have file access. Or you have some paranoid thoughts about your local system security. Btw if you somehow decrypt mail folder every time you open neomutt and encrypt it at close, it will be a good step.

## Have you tried any alternative methods?

No, but I think that the easiest option is to use encfs or gocryptfs. But this can cause serious problems if the file with the encryption information is lost.

**Comment by gahr** [comment]
Have you looked at https://neomutt.org/feature/compress?

**Comment by asakura42** [comment]
@gahr, does it work with Maildir boxes?

**Comment by gahr** [comment]
No sorry, this is only for mbox.
